app/views/layouts/application.html.erb

```diff
@@ -9,20 +9,24 @@
   </head>
   <body>
     <section id="navbar">
-      <ul>
-        <li><%= link_to "Login", new_user_session_path %></li>
-        <li><%= link_to "Sign up", new_user_registration_path %></li>
-        <li><%= link_to "Login with Facebook", user_facebook_omniauth_authorize_path %></li>
-      </ul>
+
+      <% if user_signed_in? %>
+          <li><%= link_to "Logout", destroy_user_session_path, method: :delete %></li>
+      <% else %>
+          <ul>
+            <li><%= link_to "Login", new_user_session_path %></li>
+            <li><%= link_to "Sign up", new_user_registration_path %></li>
+            <li><%= link_to "Login with Facebook", user_facebook_omniauth_authorize_path %></li>
+          </ul>
+      <% end %>
     </section>
     <section id="content">
-      <% if notice %>
-        <p><%= notice %></p>
-        <% elsif alert %>
-          <p><%= alert %></p>
-        <% end %>
-        <%= yield %>
-      </section>
+      <% flash.each do |key, value| %>
+          <%= content_tag :div, value, class: "flash #{key}" %>
```

Comment:
I can't see any css classes defined for `flash` or the keys.

Reply:
Planning to include a reference for this in upcoming PR: https://www.pivotaltracker.com/story/show/153539405

Reply:
Then add Rhode classes in that pr. Remove for now, thx.